dashboard-app/src/styles/index.css

```diff
@@ -1,11 +1,59 @@
-@import url('https://fonts.googleapis.com/css?family=Nunito:300,400,700');
+@import url('https://fonts.googleapis.com/css?family=Roboto:100,300,400');
 
 * {
-  box-sizing: border-box;
```

Comment:
How come we're removing this?

Reply:
Assumed this was added in the css reset, but it's not. Good spot